curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f3c866de7a5b784048f94b1.md

```diff
@@ -9,7 +9,7 @@ dashedName: step-36
 
 That is kind of what you want, but now it would be nice if the flavor and price were on the same line. `p` elements are <dfn>block-level</dfn> elements, so they take up the entire width of their parent element.
 
-To get them on the same line, you need to apply some styling to the `p` elements, so they behave more like <dfn>inline</dfn> elements. Add a `class` attribute with the value `item` to the first `article` element under the `Coffee` heading.
+To get them on the same line, you need to apply some styling to the `p` elements, so they behave more like <dfn>inline</dfn> elements. To do that, start by adding a `class` attribute with the value `item` to their parent `article` element.
```

Comment:
```suggestion
To get them on the same line, you need to apply some styling to the `p` elements, so they behave more like <dfn>inline</dfn> elements. To do that, start by adding a `class` attribute with the value `item` to the first  `article` element under the `Coffee` heading.
```

I don't mind the addition of `To do that, start by adding` but I don't think the last part of that sentence should be changed.

Reply:
Understood.